frontend/templates/base.html

```diff
@@ -4,18 +4,15 @@
     {% block head %}
         <meta charset="utf-8">
         <meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, user-scalable=no, width=device-width" />
-
-        <script src="/static/javascript/jquery-1.11.1.js" type="text/javascript"></script>
+        <script src="https://code.jquery.com/jquery-3.5.1.js" type="text/javascript"></script>
         <script src="/static/javascript/handlebars-v2.0.0.js" type="text/javascript"></script>
-        <script src="/static/javascript/jquery.dataTables.min.js" type="text/javascript"></script>
```

Comment:
:lady_beetle: This is a great idea, but I think `dashboard_features.js` and `patient_dashboard_features.js` still use datatables, so you'll need to add in the js/css to those pages too.

Reply:
Whoops, good catch!  Thanks- I fixed it.